Assets/Scripts/Wizzard/WizardTemplate.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu]
```

Comment:
Marcin, jak możesz to wytłumacz co ta adnotacja dodaje ?? Będziemy mogli z menu na górze Assets tworzyć template Wizzarda ? Widzę że jest to przekazywane do konstruktora wizzarda a gdzie się to wypełnia ;P? Gdzie się nadaje właściwości ??

Reply:
To jest po to by mozna bylo dodawac WizardTemplate z kontekstowego menu Unity. Wypelnia sie to poprzez dodanie recznie informacji o Wizardzie z poziomu unity. Zobacz sobie jak wyglada Witold albo Zbigniew. Mozna dodac nowych wizardow bardzo latwo, przez co testowanie i sprawdzanie roznych ustawien statystyk bedzie szybkie i proste. Tymczasowo zeby ustawic wizarda trzeba wrzucic template do turnmanagera(czy jak to sie tam nazywa) ale docelowo np do poziomu mogliby byc przypisani albo przed walka bylby ekran jak w mortalu z wyborem postaci dla siebie i dla drugiego gracza.